GliaWidgets/Lib/Coordinator/UIViewControllerCoordinator.swift

```diff
@@ -0,0 +1,75 @@
+import UIKit
+
+class UIViewControllerCoordinator: Coordinator {
```

Comment:
Since this is the base class for some Coordinators, let's ensure that it functions the way it is expected by adding tests. Then if/when internal logic will change, we will have guaranties it still functions as expected.

Reply:
Will do